src/main/kotlin/lotto/util/RandomNumberGenerator.kt

```diff
@@ -0,0 +1,14 @@
+package lotto.util
+
+import kotlin.random.Random
+
+class RandomNumberGenerator(
+    from: Int = 1,
+    until: Int = 99,
+) : NumberGenerator {
+    private val randomNumbers = List(100) { Random.nextInt(from, until) }
```

Comment:
해당 프로퍼티는 어떤 용도일까요???

Reply:
getNumbers 호출마다 임의로 생성된 숫자 리스트 중에서 6개를 반환하는데, 
getNumbers 호출마다 새로운 리스트를 생성하지 않고 이미 있는 리스트를 재활용 하고자 했습니다.

Reply:
지금 구현된 내용은 from ~ to 사이의 Random 숫자 100개를 저장하는 것 같은데요!
random 숫자들이 중복도 되기때문에 `getNumbers()` 를 호출했을때 동일한 숫자가 나올 수 있을 것 같아요.

1부터 45까지의 숫자 하나씩만 저장하는걸 원하신거였다면, 아래처럼 작성해보면 좋을 것 같아요!
```suggestion
    private val randomNumbers = (from..until).toList()
```

Reply:
아 그렇군요 반영했습니다 감사합니다!